toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/GeoViewProxy.kt

```diff
@@ -64,9 +64,21 @@ public sealed class GeoViewProxy(className: String) {
         "$className must be part of the composition when this member is called."
 
     /**
-     * True if continuous panning across the international date line is enabled in the GeoView, false otherwise.
-     * A null value represents that it is currently undetermined.
+     * True if continuous panning across the international date line is enabled, false otherwise.
+     * By default, a [GeoView] attempts to wrap the [com.arcgismaps.mapping.ArcGISMap] or [com.arcgismaps.mapping.ArcGISScene]
+     * across the international date line for a continuous panning user experience.
+     * The eastern and western hemispheres wrap to form a continuous map, giving the impression that the map is endless.
      *
+     * Wraparound display is always enabled for a [SceneView]. It is always disabled for a [LocalSceneView].
+     *
+     * To disable wraparound behavior for a composable [MapView] (or to re-enable it), you can set the composable MapView's parameter
+     * `wrapAroundMode` to the appropriate value. Wraparound can only be applied to a [MapView] if certain conditions are met, as
+     * described in [com.arcgismaps.mapping.view.WrapAroundMode.EnabledWhenSupported].
```

Comment:
we don't ship doc or source with toolkit components or the SDK, so this link will not resolve.

```suggestion
     * described in [WrapAroundMode.EnabledWhenSupported](https://developers.arcgis.com/kotlin/api-reference/arcgis-maps-kotlin/com.arcgismaps.mapping.view/-wrap-around-mode/-enabled-when-supported/index.html) in the ArcGIS Maps SDK for Kotlin documentation.
```

Reply:
Similar suggestions below for ArcGISScene.